tensorflow/compiler/mlir/tools/kernel_gen/tests/hlo_to_kernel/minimum.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: hlo_to_kernel --input=%s --output=%t --unroll_factors=4 --tile_sizes=256 --arch=sm_70
+// RUN: hlo_to_kernel --input=%s --output=%t --unroll_factors=4 --tile_sizes=256 --arch=gfx906
```

Comment:
We have dropped the support on MI50/60 https://rocm.docs.amd.com/projects/install-on-linux/en/latest/reference/system-requirements.html#supported-gpus and gfx906 upstream CI is MI200, so maybe gfx90a ?

Reply:
Should we do the same for other tests in this folder too?

Reply:
Yes please! Thanks for pointing out!